tests/utils.cpp.in

```diff
@@ -31,7 +31,7 @@
 #include <cassert>
 #include <cstring>
 
-#include "utils.h"
+#include <boost/filesystem.hpp>
```

Comment:
Are the changes to file utils.cpp.in an intentional part of this PR?

If yes, I guess perhaps because they fix a style warning found by cpplint.py?

Reply:
Yes, these changes aim to address the following warning from cpplint.py:

> behavioral-model/tools/../tests/utils.cpp:26: Found C system header after other header. Should be: utils.h, c system, c++ system, other.

We don't see this warning in our CI tests because in CI we build and test BMv2 inside a docker container but run `./tools/check_style.sh` [outside the container](https://github.com/p4lang/behavioral-model/blob/434eda7658c2fd8a8e31500e98dab9e7c1e1b0ca/.github/workflows/test.yml#L56). The auto-generated `utils.cpp` file doesn't exist outside the container.